modules/_labs/menu/react/stories/stories_visualTesting.tsx

```diff
@@ -33,11 +33,20 @@ const createMenuItems = (hasIcons?: boolean, isFocused?: boolean): StoryMenuItem
       isFocused: isFocused,
     },
     {
-      text: `Second Item with really really really long label`,
+      text: `Second Item (with a really really really long label)`,
```

Comment:
Parentheses were added to keep visual consistency with the other labels, and the "a" was added as a grammar check. 😄